### ISSUE TITLE
Reduce time command.

### ISSUE THREAD

**Issue by IceGames23** [Fefo6644/BetterJails#9]
Hello! I have a great suggestion for BetterJails for a new feature of the plugin.

Basically it's a "reduce time command" for players who are already in jail.

Command example:
`/betterjails reduce <player> <time>`

What can be done with this command? A lot of things! Example, do a system for the player in jail reduce their time doing tasks in the jail, like killing mobs, breaking blocks with a wood pickaxe, a lot of tasks can be implemented and I can make a hook with a lot of plugins for work together with BetterJails if this command exists.

Basically it's that, I hope that this command can be added in a future update, thanks for your work and time!
